runelite-client/src/main/java/net/runelite/client/plugins/boosts/BoostsConfig.java

```diff
@@ -38,15 +38,23 @@
 		NEVER
 	}
 
+	enum DisplayBoosts
+	{
+		NONE,
+		COMBAT,
+		SKILLS,
+		BOTH
+	}
+
 	@ConfigItem(
-		keyName = "enableSkill",
-		name = "Enable Skill Boosts",
-		description = "Configures whether or not to display skill boost information",
+		keyName = "displayBoosts",
+		name = "Display Boosts",
+		description = "Configures whether or not to display boost information",
```

Comment:
This should probably be updated to reflect what the setting does. `Configures what skill boosts to display`?

Reply:
Should SKILLS from the enum then be changed to NON_COMBAT?